documentation/functions/domain/LOC.md

```diff
@@ -0,0 +1,81 @@
+---
+name: LOC
+parameters:
+  - deg1
+  - min1
+  - sec1
+  - deg2
+  - min2
+  - sec2
+  - altitude
+  - size
+  - horizontal_precision
+  - vertical_precision
+parameter_types:
+  name: string
+  target: string
+  deg1: uint32
+  min1: uint32
+  sec1: float32
+  deg2: uint32
+  min2: uint32
+  sec2: float32
+  altitude: uint32
+  size: float32
+  horizontal_precision: float32
+  vertical_precision: float32
+---
+
+## Description ##
+
+Strictly follows [RFC 1876](https://datatracker.ietf.org/doc/html/rfc1876).
+
+Fairly intuitive.
```

Comment:
Not for me :)

I don’t think many of the doc pages do this but it would be nice to have a one sentence description of what this record is for and why you’d want to add it so that people don’t have to read the RFC.

Reply:
Not many providers adopted support for LOC. But in terms of RFCs, it's one of the shorter ones. If you needed it, you'd know about it. If you want to use it, read up about it.

The RFC does indicate on specific use case, however:

>    Some uses for the LOC RR have already been suggested, including the
   USENET backbone flow maps, a "visual traceroute" application showing
   the geographical path of an IP packet, and network management
   applications that could use LOC RRs to generate a map of hosts and
   routers being managed.

Maybe I'll just stick that in the LOC help. :)